src/all-exfils.ts

```diff
@@ -149,8 +170,20 @@ const ALL_EXFILS: Record<string, string[]> = {
   sandbox: ALL_DUMPED_EXFILS_FROM_SCRIPT.groundzero,
   sandbox_high: ALL_DUMPED_EXFILS_FROM_SCRIPT.groundzero,
 };
+const NEW_CUSTOM_EXFILS: Record<string, string[]> = {
+  ...ALL_NEW_CUSTOM_EXFILS,
+  bigmap: ALL_NEW_CUSTOM_EXFILS.customs,
+  rezervbase: ALL_NEW_CUSTOM_EXFILS.reserve,
+  factory4_day: ALL_NEW_CUSTOM_EXFILS.factory,
+  factory4_night: ALL_NEW_CUSTOM_EXFILS.factory,
+  tarkovstreets: ALL_NEW_CUSTOM_EXFILS.streets,
+  sandbox: ALL_NEW_CUSTOM_EXFILS.groundzero,
+  sandbox_high: ALL_NEW_CUSTOM_EXFILS.groundzero,
+};
 
 export const isValidExfilForMap = (mapName: string, exfilName: string): boolean => {
   const exfils = ALL_EXFILS[mapName] ?? [];
+  const customExfils = NEW_CUSTOM_EXFILS[mapName] ?? [];
+  exfils.push(...customExfils);
```

Comment:
```suggestion
  const vanillaExfils = ALL_EXFILS[mapName] ?? [];
  const customExfils = NEW_CUSTOM_EXFILS[mapName] ?? [];

  const exfils = {
    ...vanillaExfils,
    ...customExfils,
  };
```

everytime `isValidExfilForMap` is called, this will mutate `ALL_EXFILS` and add several time the same custom exfils.

mutations is the devil.